eclipse-scout-core/src/encoder/PlainTextEncoder.js

```diff
@@ -36,6 +36,10 @@ export default class PlainTextEncoder {
     // Separate td with ' '
     text = text.replace(/<\/td>/gi, ' ');
 
+    if (options.removeFontIcons) {
+      text = text.replace(/<span>*[ \n]+class="[^"]*font-icon[^"]*">[^<]*<\/span>/gmi, '');
```

Comment:
Is the first ">" correct?
E.g. use \s instead of [ \n]?

Reply:
No, it is not correct😁 thx. \s would be better as well